docs/SPEC.md

```diff
@@ -202,7 +199,7 @@ Examples:
     1h15m  // 1 hour and 15 minutes
     5w
     1mo5d  // 1 month and 5 days
-    -1mo5d // negative 1 month and 5 days
+    -1y    // negative 1 year
```

Comment:
Is there a reason to change this line?

Reply:
Ah the reason was because I didn't want to mix the duration units and then the PR changed so I'll change this back.